source/apiVolontaria/volunteer/models.py

```diff
@@ -219,6 +227,11 @@ class Participation(models.Model):
     (M2M relationship) and store informations concerning that association.
 
     """
+    PRESENCE_STATUS = (
+        ('I', u'Initialisation'),
+        ('A', u'Absent'),
+        ('P', u'Présent'),
+    )
```

Comment:
Can you use I18N function on these description ?
(The second part of the tuple only)